client/components/data/query-rewind-backup-status/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { delay } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getRewindBackupProgress } from 'state/activity-log/actions';
+
+class QueryRewindBackupStatus extends PureComponent {
+	static propTypes = {
+		freshness: PropTypes.number,
+		queryDelay: PropTypes.number.isRequired,
+		downloadId: PropTypes.number,
+		siteId: PropTypes.number.isRequired,
+	};
+
+	static defaultProps = {
+		queryDelay: 0,
+	};
+
+	query( props ) {
+		const { getBackupProgress, queryDelay, downloadId, siteId } = props;
+		if ( ( siteId, downloadId ) ) {
```

Comment:
was the use of the comma operator here intentional?

Reply:
It's updated now to check siteId and downloadId.